src/System.Management.Automation/FormatAndOutput/common/FormattingObjects.cs

```diff
@@ -207,6 +207,7 @@ internal sealed partial class TableColumnInfo : FormatInfoData
         public int alignment = TextAlignment.Left;
         public string label = null;
         public string propertyName = null;
+        internal bool HeaderMatchesProperty = true;
```

Comment:
Why not just `public`?

Reply:
I guess if we want to follow the other members, there's no reason not to make it public